plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -2,6 +2,7 @@
 
 {-# OPTIONS -fno-warn-missing-pattern-synonym-signatures #-}
 
+{-# LANGUAGE BlockArguments        #-}
```

Comment:
Do we really need `BlockArguments`?  I don't recall coming across it before, but it doesn't appear to be doing much for us here (although I didn't try removing it). It looks as if it's being used in the `Parsable` instance below, but I don't think that's changed in this PR; maybe I'm missing something.

Reply:
It's so that I can write `... >>= oneShot \case ...` instead of `... >>= oneShot (\case ...)`.